appveyor.yml

```diff
@@ -0,0 +1,41 @@
+environment:
+    ANDROID_HOME: "C:\\android-sdk-windows"
+    ANDROID_NDK: "C:\\android-sdk-windows\\android-ndk-r10e"
+    ANDROID_BUILD_VERSION: 26
+    ANDROID_TOOLS_VERSION: 26.0.3
+
+    GRADLE_OPTS: -Dorg.gradle.daemon=false
+
+    SDK_TOOLS_URL: https://dl.google.com/android/repository/sdk-tools-windows-3859397.zip
+    NDK_TOOLS_URL: https://dl.google.com/android/repository/android-ndk-r10e-windows-x86_64.zip
+
+    matrix:
+        - nodejs_version: 8
+        - nodejs_version: 10
+
+install:
+    # Install Android SDK Tools
+    - mkdir "%ANDROID_HOME%"
+    - appveyor DownloadFile "%SDK_TOOLS_URL%" -FileName "%TMP%/sdk-tools.zip"
+    - 7z x "%TMP%/sdk-tools.zip" -o"%ANDROID_HOME%" > nul
+    - appveyor DownloadFile "%NDK_TOOLS_URL%" -FileName "%TMP%/ndk.zip"
+    - 7z x "%TMP%/ndk.zip" -o"%ANDROID_HOME%" > nul
+    - set PATH=%PATH%;"%ANDROID_HOME%\tools\bin"
+
+    - yes 2> nul | sdkmanager --licenses > nul
+    - sdkmanager "system-images;android-19;google_apis;armeabi-v7a"
+    - sdkmanager "platforms;android-26"
+    - sdkmanager  "build-tools;26.0.3"
+    - sdkmanager "add-ons;addon-google_apis-google-23"
+    - sdkmanager "extras;android;m2repository"
+
+    - ps: Install-Product node $env:nodejs_version
+    - npm install
```

Comment:
I suggest doing `appveyor-retry npm` to make it resilient against intermittent network issues. I added this in the react-native-windows AppVeyor configuration to eliminate a not-uncommon source of flakiness. https://github.com/Microsoft/react-native-windows/blob/master/appveyor.yml#L40

You may also want to add `appveyor-retry npm install npm@lts`.